01_Tutorial/01_Basics.md

```diff
@@ -25,29 +29,43 @@ You will need to get the code for OrbitDB and its dependency, IPFS, and make it
 
 #### In Node.js
 
-Choose a project directory and `cd` to there from your command line. Then run the following command:
+Create a project directory (for example `orbitdb-ch1`) and `cd` to there from your command line. Then run the following command:
 
 ```bash
 $ npm init --yes
-$ npm install --save orbit-db ipfs
+$ npm install orbit-db ipfs
 ```
+If you want to use Git to track your progress, we also suggest typing the following command lines while in the folder:
 
-This will create a `package.json`, `package-lock.json`, and `node_modules` folder.
-
-> **Note:** If you're running on a Windows prompt, or if you don't have certain build tools like [`g++`](https://gcc.gnu.org) and [`python`](https://www.python.org) installed, you may see a noisy console output with lots of warnings and errors. Keep going, your code should still run.
-
-> **Note:** Adding the `--yes` flag will automatically use your npm defaults. You can go through and edit the package.json later, but it's not entirely necessary for this part of the tutorial.
-
-If you want to use Git to track your progress, we also suggest the following:
+```bash
+$ git init
+$ echo node_modules > .gitignore
+```
 
+The output will be
 ```bash
+$ mkdir orbitdb-ch1
+$ cd orbitdb-ch1
 $ git init
+   Initialized empty Git repository in /Users/myPath/orbitdb-ch1/.git/
 $ echo node_modules > .gitignore
+$ npm init --yes
+   Wrote to /Users/myPath/orbitdb-ch1/package.json:
+   ... some stuff...
+$ npm install orbit-db ipfs
+   ... a lot more stuff...
+   + orbit-db@0.26.1
+   + ipfs@0.55.4
+   added 1165 packages from 1305 contributors and audited 1165 packages in 174.258s
+
 ```
+This will create 2 files `package.json`, `package-lock.json`, and a `node_modules` folder.
 
-Of course, be careful before copying and pasting any commands anyone ever tells you into your terminal. If you don't understand a command, figure out what it is supposed to do, before copying it over. Copy and paste at your own risk.
+> **Note:** If you're running on a Windows prompt, or if you don't have certain build tools like [`g++`](https://gcc.gnu.org) and [`python`](https://www.python.org) installed, you may see a noisy console output with lots of warnings and errors. Keep going, your code should still run.
```

Comment:
If it's possible in Github, would you mind doing it? I don't know Github enough for that. I'm a true beginner, no kidding.

Reply:
I think it might be possible to reduce that security advisory to:
> If you copy something into your terminal, be sure you  know what it is doing.
> Copy and paste at your own risk.